csrankings-l.csv

```diff
@@ -447,6 +447,7 @@ Liang Wang 0001,Chinese Academy of Sciences,http://www.cbsr.ia.ac.cn/users/liang
 Liang Wang 0006,Nanjing University,http://cs.nju.edu.cn/wangliang,O6EwhtYAAAAJ
 Liang Zhao 0002,Emory University,https://cs.emory.edu/~lzhao41,qnvyqtwAAAAJ
 Liang Zheng 0001,Australian National University,https://cs.anu.edu.au/people/liang-zheng,vNHqr3oAAAAJ
+Liang-Jie Zhang,Shenzhen University,https://orcid.org/0000-0002-6219-0853,DkEKl2QAAAAJ

```

Comment:
OrcID page not acceptable as home page.

Reply:
Hi, His ieeexplore home page is OK ? 
https://ieeexplore.ieee.org/author/37280178900

Reply:
No, it looks thoroughly broken (read it). Please point to an institutional page.

Reply:
I get a home page from csse.szu.edu.cn (College of Computer Science and Software Engineering website).
https://csse.szu.edu.cn/en/pages/user/index?id=1298

Thanks for your review.